FSharp.AutoComplete/Program.fs

```diff
@@ -521,11 +545,11 @@ module internal Main =
               let declstrings =
                 [ for tld in decls do
                     let m = tld.Declaration.Range
-                    let (s1, e1), (s2, e2) =  ((m.StartColumn, m.StartLine), (m.EndColumn, m.EndLine))
+                    let (s1, e1), (s2, e2) =  ((m.StartColumn + 1, m.StartLine), (m.EndColumn + 1, m.EndLine))
                     yield sprintf "[%d:%d-%d:%d] %s" e1 s1 e2 s2 tld.Declaration.Name
                     for d in tld.Nested do
                       let m = d.Range
-                      let (s1, e1), (s2, e2) = ((m.StartColumn, m.StartLine), (m.EndColumn, m.EndLine))
+                      let (s1, e1), (s2, e2) = ((m.StartColumn + 1, m.StartLine), (m.EndColumn + 1, m.EndLine))
                       yield sprintf "  - [%d:%d-%d:%d] %s" e1 s1 e2 s2 d.Name ]
               printAgent.WriteLine(sprintf "DATA: declarations\n%s\n<<EOF>>" (String.concat "\n" declstrings))
           | Json -> prAsJson { Kind = "declarations"; Data = decls }
```

Comment:
OTOH, maybe we should have a record type for this as well. It'd be nice to have StartColumn/StartLine etc in this type as well, for consistency.